bucket/f3d.json

```
@@ -0,0 +1,30 @@
+{
+    "version": "1.2.1",
+    "description": "F3D, a fast and minimalist 3D viewer",
+    "homepage": "https://f3d-app.github.io/f3d/",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/f3d-app/f3d/releases/download/v1.2.1/f3d-1.2.1-win64.zip",
+            "hash": "9258c564be22a7a4a8f0ccada0f4ddfc257e09a8cda92eef8b440b1c0fb1ef15"
+        }
+    },
+    "extract_dir": "f3d-1.2.1-win64",
+    "bin": [
+        [
+            "bin\\f3d.exe",
+            "bin\\f3d"
+        ]
+    ],
+    "shortcuts": [
+        [
+            "f3d.exe",
+            "f3d"
+        ]
+    ],
+    "persist": "config.json",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/f3d-app/f3d/releases/download/v$version/f3d-$version-win64.zip"
+    }
+}
```

```suggestion
}

```

add a new line at the end of file. (otherwise CI will fail)